app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,20 @@
+class CommentsController < ApplicationController
+  def new
+    @user = current_user
+    @post_id = params.require('id')
+  end
+
+  def create
+    data = params.require('data')
+    post_id = params.require('id')
```

Comment:
[OPTIONAL] It is better to create strong parameters in a  private method. It is more secure and reduces repetition. Kindly review this [lesson](`https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides`). Also you are not creating the strong parameters properly since you are meant to require and permit certain parameters.